### PR TITLE
Deliveryboy

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Adds Delivery Boy uniform as an alternative uniform for cargo (UPS reference)

## Why It's Good For The Game
Alternative uniform, cargo didnt have any before

## Images of changes
image

## Testing
Tested

## Changelog
:cl:
add: New uniform for cargo - Delivery Boy uniform
/:cl:

### CODE REVIEW

File: code/modules/client/preference/loadout/loadout_hat.dm [hunk 1]
```diff
@@ -77,6 +77,12 @@
 	path = /obj/item/clothing/head/soft/sec
 	allowed_roles = list("Head of Security", "Warden", "Security Officer")
 
+/datum/gear/hat/deliverysoft
+	display_name = "Delivery Boy cap"
+	path = /obj/item/clothing/head/soft/deliverysoft
+	allowed_roles = list("Quartermaster","Cargo Technician")
+
+
```
Comment (Henri215) [inline code/modules/client/preference/loadout/loadout_hat.dm:85]:
```suggestion
	allowed_roles = list("Quartermaster", "Cargo Technician")

```

File: code/modules/client/preference/loadout/loadout_uniform.dm [hunk 2]
```diff
@@ -232,6 +232,14 @@
 	path = /obj/item/clothing/under/rank/medical/scrubs/green
 	allowed_roles = list("Chief Medical Officer", "Medical Doctor")
 
+/datum/gear/uniform/cargo
+	main_typepath = /datum/gear/uniform/cargo
+
+/datum/gear/uniform/cargo/delivery
+	display_name = "Delivery Boy Uniform"
+	path = /obj/item/clothing/under/rank/cargo/deliveryboy
+	allowed_roles = list("Quartermaster","Cargo Technician")
```
Comment (Henri215) [inline code/modules/client/preference/loadout/loadout_uniform.dm:241]:
```suggestion
	allowed_roles = list("Quartermaster", "Cargo Technician")
```

File: code/modules/clothing/head/soft_caps.dm [hunk 3]
```diff
@@ -103,6 +106,13 @@
 	item_color = "rainbow"
 	dog_fashion = /datum/dog_fashion/head/softcap
 
+/obj/item/clothing/head/soft/deliverysoft
+	name = "delivery boy cap"
+	desc = "It's a baseball hat that is part of delivery boy unifrom."
```
Comment (Henri215) [inline code/modules/clothing/head/soft_caps.dm:111]:
```suggestion
	desc = "It's a baseball hat that is part of the delivery boy uniform."
```

File: code/modules/clothing/under/jobs/cargo.dm [hunk 4]
```diff
@@ -16,6 +16,13 @@
 	item_state = "lb_suit"
 	item_color = "qm"
 
+/obj/item/clothing/under/rank/cargo/deliveryboy
+	name = "delivery boy uniform"
+	desc = "It's a jumpsuit worn by cargo delivery boy."
```
Comment (Henri215) [inline code/modules/clothing/under/jobs/cargo.dm:21]:
```suggestion
	desc = "It's a jumpsuit worn by the cargo delivery boy."
```

**Review by Henri215 — COMMENTED** [review]
image

Remove that empty line from the changelog

**Comment by McRamon** [comment]
@Henri215 thanks, everything fixed

**Comment by Venuska1117** [comment]
hmm... its only me or vox front hat sprite look like he have .. uh... pillow on head (?)

File: code/modules/client/preference/loadout/loadout_uniform.dm
Comment (Christasmurf) [inline code/modules/client/preference/loadout/loadout_uniform.dm:1]: Generally for loadout items you do it like "Uniform, delivery boy" for the display name. Same with the cap, "Cap, delivery boy"

**Comment by Probablyjay19** [comment]
Me when I work a long day of manual labor in my awesome truck only to do it but unpaid and without the awesome truck this time in funny space man game: 

image

**Comment by Henri215** [comment]
image

This vox row of pixels is supposed to be darker than the others?

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]
conflicted for 3 weeks please deconflict for merge

**Pull request merged**